examples/dt/uni02beta/control-plane/service-values.yaml

```diff
@@ -0,0 +1,115 @@
+---
+apiVersion: v1
+kind: ConfigMap
+
+metadata:
+  name: service-values
+  annotations:
+    config.kubernetes.io/local-config: "true"
+
+data:
+  cinder:
+    extraMounts:
+      - extraVol:
+          - propagation:
+              - CinderVolume
+            extraVolType: Undefined
+            volumes:
+              - name: nfs-shares
+                projected:
+                  sources:
+                    - secret:
+                        name: cinder-volume-ontap-shares-secrets
+            mounts:
+              - name: nfs-shares
+                mountPath: /etc/cinder/nfs_shares.d
+                readOnly: true
+
+  cinderVolumes:
+    ontap-nfs:
+      customServiceConfig: |
+        [ontap]
```

Comment:
I'll soon post a PR to update cinder's backend samples that will add these lines to the ontap configuration, so you should add them too. I recommend adding them after L33.

```
            nfs_snapshot_support=true
            nas_secure_file_operations=false
            nas_secure_file_permissions=false
```